components/automl/prediction_service_batch_predict/component.py

```diff
@@ -0,0 +1,55 @@
+from typing import NamedTuple
+
+
+def automl_prediction_service_batch_predict(
+    model_path,
```

Comment:
This is more of a general questions going to back to Lint rules, @numerology are we planning to use 2 space or 4?